internal/build/build.go

```diff
@@ -0,0 +1,224 @@
+// Package build exposes functions to build images from source code.
+package build
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"text/template"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	ketchv1 "github.com/shipa-corp/ketch/internal/api/v1beta1"
+	"github.com/shipa-corp/ketch/internal/archive"
+	"github.com/shipa-corp/ketch/internal/docker"
+	"github.com/shipa-corp/ketch/internal/errors"
+)
+
+const (
+	archiveFileName      = "archive.tar.gz"
```

Comment:
We define it [here](https://github.com/shipa-corp/ketch/pull/50/files#diff-ebba829eab1ca7dd99d66964c8eddd4febfda8f9dd35dbcbd28e78cece01d37bR15). Is it possible to define such constants once and reuse them?